services/orchest-webserver/client/src/components/ParameterEditor.tsx

```diff
@@ -59,6 +60,13 @@ const ParameterEditor: React.FC<IParameterEditorProps> = (props) => {
     }
   }, []);
 
+  const codeMirrorRef = React.useRef<CodeMirror | null>(null);
+  React.useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    if (activeParameter) codeMirrorRef.current?.editor.focus(); // `editor` was not defined.
```

Comment:
Maybe it's better to cast to any here?

```suggestion
    if (activeParameter) (codeMirrorRef.current? as any).editor.focus();
```

That way you don't need the `@ts-ignore` at least.

Reply:
Pushed my change. The only difference is that the optional operator should be left out.

```
// invalid syntax
(codeMirrorRef.current? as any).editor.focus();
// valid syntax
(codeMirrorRef.current as any).editor.focus();
```